### PR TITLE
Encodings: Add readme.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-specs/issues/243.

### CODE REVIEW

File: encodings/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+This documentation serves to document the on-the-wire encoding format supported in
+OpenCensus. It describes the key types and the overall behavior.
+
+## Formats
+* [Binary Encoding](BinaryEncoding.md)
+  * [TraceContext Binary Encoding](BinaryEncoding.md#trace-context)
+  * [TagContext Binary Encoding](BinaryEncoding.md#tag-context)
+  * [Census Server Stats Encoding](CensusServerStatsEncoding.md)
+
+* Text Encoding
```
Comment (bogdandrutu) [inline encodings/README.md]: Http Encoding :)

Comment (songy23) [reply encodings/README.md]: SG, updated.

File: encodings/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# OpenCensus Library Encoding Package
+This documentation serves to document the on-the-wire encoding format supported in
+OpenCensus. It describes the key types and the overall behavior.
+
+## Formats
+* [Binary Encoding](BinaryEncoding.md)
```
Comment (SergeyKanzhelev) [inline encodings/README.md]: btw, we are working towards making it a standard https://github.com/w3c/trace-context-binary/pull/2 I'd appreciate your review on that PR

**Comment by songy23** [comment]
Thanks!

**Pull request merged**